cmd/swapd/main.go

```diff
@@ -284,7 +296,7 @@ func (d *daemon) make(c *cli.Context) error {
 
 	// By default, the chain ID is derived from the `flagEnv` value, but it can be overridden if
 	// `flagEthereumChainID` is passed:
-	if c.Uint(flagEthereumChainID) != 0 {
+	if c.IsSet(flagEthereumChainID) {
```

Comment:
I'll remove `flagEthereumChainID` in a follow-up PR in favor of dynamic determination of the chain ID.